src/expertsystem/reaction/topology.py

```diff
@@ -34,8 +34,8 @@
 class Edge:
     """Struct-like definition of an edge, used in `Topology`."""
 
-    ending_node_id: Optional[int] = attr.ib(default=None)
     originating_node_id: Optional[int] = attr.ib(default=None)
+    ending_node_id: Optional[int] = attr.ib(default=None)
```

Comment:
Note: I thought ending node was put first, because of common notation (think of _M\_fi_). I also remember there were some implementation issues where it was more convenient to put ending node first, because initial state is usually just one edge (can't remember where exactly).

Reply:
It just seems counter intuitive to me. Yeah the amplitude M_fi is written "reverse", but I think that's a separate topic. When building a topology and I want to make a connection from node 0 to node 1 (note that the edges have direction), then I want to write this in the form `Edge(0,1)`. Currently it was `Edge(1,0)`.

Reply:
Ok. I initially also preferred originating node first, but I remember there was some reason not to do so. It's indeed better if originating node first works find in the current implementation.